### PR TITLE
docs: update release note docs

### PR DESCRIPTION
**Pull request by cdrage**
docs: update release note docs

### What does this PR do?

* Updates the release note docs with minor changes encountered as a result of doing another release.
* Adds a template for blog posting.

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

N/A

### How to test this PR?

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Review by lstocchi — APPROVED** [review]
LGTM

File: RELEASE.md [hunk 1]
```diff
@@ -2,31 +2,33 @@
...
 1. Go to https://github.com/containers/podman-desktop/actions/workflows/release.yaml
 1. Click on the top right drop-down menu `Run workflow`
 1. Enter the name of the release. Example: `0.12.0` (DO NOT use the v prefix like v0.12.0)
 1. Specify the branch to use for the new release. It's main for all major releases. For a bugfix release, you'll select a different branch.
-1. Click on the `Run workflow` button
+1. Click on the `Run workflow` button. 
+1. Note: `Run workflow` takes approximatley 30-50 minutes. Brew a coffee, work on the release notes, and/or complete the next two steps while you wait.
 1. Close the milestone for the respective release, make sure that all tasks within the milesetone are completed / updated before closing. https://github.com/containers/podman-desktop/milestones
-1. If not already created, create a new milestone for the upcoming release after for example `0.13.0` and click on `Create milestone`
+1. Click on `New Milestone` and create a new milestone for the NEXT release.
```
Comment (jeffmaury) [inline RELEASE.md]: It might have been already created, why removing this case ?

File: RELEASE.md [hunk 2]
```diff
@@ -2,31 +2,33 @@
...
 1. Check that https://github.com/containers/podman-desktop/actions/workflows/release.yaml has been completed. Sometimes it will flake, so you may need to re-run it.
-1. There should be an automated PR that has been created. Approve PR for the upcoming version. The title looks like `chore: 📢 Bump version to 0.13.0`
+1. There should be an automated PR that has been created. Approve this and set to auto-merge. This will be automatically merged in after all tests have been ran (takes 10-30 minutes). The title looks like `chore: 📢 Bump version to 0.13.0`.
+1. Above PR MUST be merged before continuing with the steps.
 1. Edit the new release https://github.com/containers/podman-desktop/releases/edit/v0.12.0
 1. Select previous tag (v0.11.0) and click on `Generate release notes` and the click on `Update release`
 
-## Package manager workflow
+## Updating packages
```
Comment (jeffmaury) [inline RELEASE.md]: packages seems vague to me, may be adding on various managers ?

File: RELEASE.md [hunk 3]
```diff
@@ -90,24 +92,34 @@ You can view the progress at: https://community.chocolatey.org/packages/podman-d
 1. Create a PR to the repository with a title like `feat: bump to v0.11.0`
 1. If the PR passes all tests, merge the PR
 
-## podman.io website
+## Documentation
 
-Publish update to the podman website at https://podman.io
+## Release notes
```
Comment (jeffmaury) [inline RELEASE.md]: I think one # is missing

**Pull request merged**